### PR TITLE
fix indentation and comments in main.js

### PR DESCRIPTION
**Pull request by renrith**
- Comments translated to English
- Added consistent formatting
- Configured ESlint to allow for tab-based indentation
- Removed a couple of unnecessary variable declarations

### CODE REVIEW

File: _src/assets/js/main.js [hunk 1]
```diff
@@ -126,109 +120,109 @@ checkboxMontserratSelector.addEventListener('click', chooseFont);
...
+/* fill section functionality */
 
-/* Name field */
+// name field
 
 const fillNameSelector = document.querySelector('#full-name');
 
 
-fillNameSelector.addEventListener('keyup', function(e){
+fillNameSelector.addEventListener('keyup', function (e) {
```
Comment (natalia-pn) [inline _src/assets/js/main.js]: I'd keep the brackets next to the word "function".

File: _src/assets/js/main.js [hunk 2]
```diff
@@ -126,109 +120,109 @@ checkboxMontserratSelector.addEventListener('click', chooseFont);
...
     cardNameSelector.innerHTML = writer.value;
 });
 
-//////////occupation field///////////
+// occupation field
 
 const fillOccupationSelector = document.querySelector('#occupation');
 
-fillOccupationSelector.addEventListener('keyup', function(e){
+fillOccupationSelector.addEventListener('keyup', function (e) {
```
Comment (natalia-pn) [inline _src/assets/js/main.js]: Same here: function(e) {

File: _src/assets/js/main.js [hunk 3]
```diff
@@ -126,109 +120,109 @@ checkboxMontserratSelector.addEventListener('click', chooseFont);
...
-const li__phone = document.querySelector('.li__phone');
-const li__linkedin = document.querySelector('.li__linkedin');
-const li__github = document.querySelector('.li__github');
+const liEmail = document.querySelector('.li__email');
+const liPhone = document.querySelector('.li__phone');
+const liLinkedin = document.querySelector('.li__linkedin');
+const liGithub = document.querySelector('.li__github');
 
-fillEmailSelector.addEventListener('keyup', function(e){
+fillEmailSelector.addEventListener('keyup', function (e) {
```
Comment (natalia-pn) [inline _src/assets/js/main.js]: function(e) {

File: _src/assets/js/main.js [hunk 4]
```diff
@@ -126,109 +120,109 @@ checkboxMontserratSelector.addEventListener('click', chooseFont);
...
 });
 
 
-    ////phone  number field////
+//phone number field
 
 const fillPhoneSelector = document.querySelector('#phone');
 
-fillPhoneSelector.addEventListener('keyup', function(e){
+fillPhoneSelector.addEventListener('keyup', function (e) {
```
Comment (natalia-pn) [inline _src/assets/js/main.js]: function(e) {

File: _src/assets/js/main.js [hunk 5]
```diff
@@ -126,109 +120,109 @@ checkboxMontserratSelector.addEventListener('click', chooseFont);
...
 
 
-   ///LinkedIn field////
+//LinkedIn field
 
 const fillLinkedInSelector = document.querySelector('#linkedin');
 
-fillLinkedInSelector.addEventListener('keyup', function(e){
-    const writer = e.currentTarget; 
+fillLinkedInSelector.addEventListener('keyup', function (e) {
```
Comment (natalia-pn) [inline _src/assets/js/main.js]: function(e) {

File: _src/assets/js/main.js [hunk 6]
```diff
@@ -126,109 +120,109 @@ checkboxMontserratSelector.addEventListener('click', chooseFont);
...
+    liLinkedin.innerHTML = `<a href="https://www.linkedin.com/in/${writer.value}"><div class="social-icon social-icon--green icon__linkedin"><span class="fab fa-linkedin-in"></span></div></a>`;
 });
 
-     ////Github field///
+//Github field
 
 const fillGithubSelector = document.querySelector('#github');
 
-fillGithubSelector.addEventListener('keyup', function(e){
+fillGithubSelector.addEventListener('keyup', function (e) {
```
Comment (natalia-pn) [inline _src/assets/js/main.js]: function(e) {

**Comment by renrith** [comment]
That's exactly what I wanted to do, but when I remove the spaces linter recognizes them as errors.

In any case, I'll change those back to how they were and merge to dev since we're running out of time.

**Pull request merged**